### ISSUE TITLE
[1.4.7] Publish release version of Equals.Fody instead of debug

### ISSUE THREAD

**Issue by GeertvanHorrik** [Fody/Equals#25]
My store certification just failed because:

```
Debug configuration
Error Found: The debug configuration test detected the following errors:
The binary Equals.dll is built in debug mode.
Impact if not fixed: Windows Store doesn’t allow a debug version of an app.
How to fix: Please make sure the app isn’t linking to any debug versions of a framework and it is built with release configuration with optimization enabled. If this is a managed app please make sure you have installed the correct version of .NET framework.
```

I know that Equals.dll shouldn't even be in the package, but why is it a debug version anyway?

**Comment by rjasica** [comment]
My bad, 1.4.8 is ready.

**Comment by GeertvanHorrik** [comment]
I think you made the same mistake with 1.4.9?

**Comment by rjasica** [comment]
fixed in 1.4.10
